setup.py

```diff
@@ -59,13 +59,13 @@
         ]
 
 DEPENDENCIES = [
-        "numpy",
-        "pandas",
-        "pandas-datareader>=0.1.1",
-        "matplotlib",
-        "lxml", 
+        "numpy=1.10.4",
```

Comment:
i was kind of split on this one too. On one hand back in 'you know where' we used to freeze version numbers for our packages. Maybe for this it wouldn't make sense to do so. I can roll these back